src/client/Client.cc

```diff
@@ -145,6 +145,10 @@
 
 #define DEBUG_GETATTR_CAPS (CEPH_CAP_XATTR_SHARED)
 
+#ifndef S_IXUGO
+#define S_IXUGO	(S_IXUSR|S_IXGRP|S_IXOTH)
+#endif
```

Comment:
Do you really need the above bit? This is defined on Linux in include/linux/stat.h. Do you expect that it won't be defined on some exotic arches? (Windows?)

Reply:
Yeah, but it wasn't defined in below version of the linux. Hence I added that.

# uname -a
Linux kotresh-T490s 5.4.0-70-generic #78~18.04.1-Ubuntu SMP Sat Mar 20 14:10:07 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux

Reply:
Really? That's a fairly standard header file on most Linux distros. Was your Ubuntu box missing some packages, perhaps?

Reply:
I don't think so. It's standard ubuntu 18.04 LTS.

Reply:
Ok. It's unlikely to harm anything so I won't object to it. That definition is not new though and ought to be in standard headers for most distros.